src/styles/index.css

```diff
@@ -0,0 +1,521 @@
+@font-face {
+  font-family: FuturaPT;
+  src:
+    local(futurapt-book),
+    url(./fonts/futura-pt-book.otf) format("opentype");
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+}
+
+@font-face {
+  font-family: Montserrat;
+  src:
+    local(montserrat-regular),
+    url(./fonts/Montserrat-Regular.ttf) format("truetype");
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+}
+
+@font-face {
+  font-family: Montserrat;
+  src:
+    local(montserrat-bold),
+    url(./fonts/Montserrat-Bold.ttf) format("truetype");
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+}
+
+@font-face {
+  font-family: PlayfairDisplay;
+  src:
+    local(playfairdisplay-regular),
+    url(./fonts/PlayfairDisplay-Regular.ttf) format("truetype");
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+}
+
+@font-face {
+  font-family: ProximaNova;
+  src:
+    local(proximanova-regular),
+    url(./fonts/ProximaNova-Regular.ttf) format("truetype");
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+}
+
+@font-face {
+  font-family: RobotoSlab;
+  src:
+    local(robotoslab-bold),
+    url(./fonts/roboto-slab-bold.ttf) format("truetype");
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+}
+
+@font-face {
+  font-family: SourceSerifPro;
+  src:
+    local(sourceserifpro-bold),
+    url(./fonts/source-serif-pro-bold.ttf) format("truetype");
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+}
+
+:root {
+  --bubble-gum-pink: #ff5ab7;
+  --pinkish-grey: #c4c4c4;
+  --black: #000;
+  --black-5: rgba(0, 0, 0, 0.05);
+  --pale-grey: #f5f6fa;
+  --azure: #0290ff;
+  --medium-blue: #307bbe;
+  --dodger-blue: #379cf6;
+  --white: #e8e8e8;
+}
+
+.Text-Style-5 {
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--azure);
+}
+
+.Text-Style-6 {
+  font-family: RobotoSlab, sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
```

Comment:
- [ ] It would be so much better if you delete the styles that you are not using. This will make your code looks messy and is unnecessary code that can slow down your app. Kindly delete the code that you are not using. 
`Text-Style-6`, `Text-Style-10`, `Text-Style-2`, `Text-Style-4`, `Text-Style-8`, `Text-Style-9`, `Text-Style-3` 

🛠️ 👍

Reply:
Thanks so much for pointing that out, @BrianSammit! You are totally right! :pray: